internal/handlers/handler.go

```diff
@@ -236,6 +238,18 @@ func (h *Handler) msgStorage(ctx context.Context, msg *wire.OpMsg) (common.Stora
 		return nil, lazyerrors.Errorf("Handler.msgStorage: %w", err)
 	}
 
+	if !tableExist {
```

Comment:
`db.foo.find()`should not create a table as it doesn't create a collection in MongoDB. It should be created on `insert` only.

Reply:
I removed this change, but we still need to fix this behavior, because on MongoDB, you can query/count documents on a non-existing collection and it handles that gracefully by returning empty results. 

We might need to catch the `relation "non_existing_collection" does not exist (SQLSTATE 42P01)` error and return empty results.